### PR TITLE
describe REPL with sample session and refer to blog posts

### LINKED ISSUE #303: Suggestion: highlight repl possibilites
Hi!

I'm a quite happy user of JVM-backed cljs with Lein. I go for that since it's what I know, works well, has the most plugins etc.

The only weak spot is when I want a repl NOW for generic stuff. Normally I'd have to

* go to a (properly setup) clojurescript project
* run some command (or even start Emacs)
* wait for JVM startup
* wait for cljs compilation of my whole project

That sucks if all I want is to show `(* 3 2)` to my friend.

So http://clojurescript.io/ is my go-to cljs repl. That's not ideal either - startup could be faster, I don't like the autocompletions / parens matching, etc.

Long story short, I think it'd be be a great value proposition if Lumo offers an instant, project-agnostic repl.

Personally I don't know Lumo much, I suspect it could offer that possibility but it's not highlighted in the README either.

What I'd suggest is:

- make sure lumo offers a repl you can fire up anywhere in less than a second (similar to ruby `irb`).
- make sure getting that comes for default for users, e.g. I `brew install lumo` and already have this instant-generic-repl prepared for me - zero config needed
- Highlight it in the readme!

I'm not asking for this in a whimsical way. I just suggest what could be a killer use case which could skyrocket Lumo's usage.

Cheers - Victor

### PR DESCRIPTION
**Pull request by penryu**
Fixes #303. Expands README to include mention of the REPL, and provides a sample REPL session to drive the point home.

### CODE REVIEW

**Comment by arichiardi** [comment]
LGTM

**Comment by anmonteiro** [comment]
Happy to add an example REPL session to the README, but I feel like an interop example with some crypto stuff might feel too foreign to potentially interested folks.

**Comment by penryu** [comment]
I don't disagree. I was hoping it would accentuate the ease of interop with the node ecosystem in a few lines.

I'll look at other npm options.

**Comment by anmonteiro** [comment]
> I'll look at other npm options.

`left-pad` is a classic.

**Comment by penryu** [comment]
Granted, it's not as... _basic_ as a left-pad demo, but not as sinister as bcrypt. `express` and `request` feel pretty accessible, and having them talk to each other in half a dozen lines has a nice dynamic feel.

But if it still feels noisy, I can whittle it down a bit more.

**Comment by penryu** [comment]
Ping.

I see there's an appveyor failure, but it looks unrelated to this doc change. Is there anything way I should improve this PR further?

File: README.md [hunk 1]
```diff
@@ -85,9 +85,34 @@ $ docker run -it anmonteiro/lumo
 
 ## Using Lumo
 
-Enter `lumo` at the command line to launch it.
+### Interactive ClojureScript REPL
+
+Enter `lumo` at the command line to launch the ClojureScript REPL.
+
+```clojure
+$ npm init -y && npm i express request request-promise
```
Comment (arichiardi) [inline README.md]: maybe this one can become `npm install express...` just for clarity

**Review by arichiardi — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (penryu) [reply README.md]: Never a bad idea. Done in 1739274.

**Comment by penryu** [comment]
Deepest apologies for the force push. There was some metadata that I didn't want included.

**Comment by arichiardi** [comment]
@anmonteiro, this seems ok, may I push the button?

**Comment by penryu** [comment]
Thank you for the opportunity to contribute! I appreciate it!

**Pull request merged**